### ISSUE TITLE
ATT drop the internet when using eap_proxy?

### ISSUE THREAD

**Issue by ryc111** [pyther/goeap_proxy#4]
Hi Matthew, 

First, thank you for your amazing tool!
It works well and very stable, stayed online for more than a year, until today. 

Starting this morning, this seems not work. I connected the ATT modem behind my router. Router gets online, but few minutes later it lost all connections. 

Do you also have this issue?

**Comment by ryc111** [comment]
Maybe this is due to ATT try to do something.
And I just realized that goeap binary was compiled back in May 2020. So I pulled and built new binary.
Now the proxy works fine for few hours. Will see if this will reproduce later.

**Comment by pyther** [comment]
If it helps track down your issue, I haven't had any problems with my AT&T service. If you continue to see problems pay close attention to the type of EAP packets that are being proxied (or lack thereof).

**Comment by ryc111** [comment]
I tried several ways this Wednesday to fix the issue.

In sequence:
1. connect the AT&T modem to my router. and restart my router.
2. connect AT&T modem directly to ONT and serve network (The network was down in the middle of the meeting.).
3. few hours later, rebuilt the goeap binary from head, cp to Openwrt router, wire back to the proxy mode. restart whole system. proxy works ( Now the network up for 36 hours )

Possible cause:
The AT&T modem model is Arris BGW210-700 and I realized that the modem upgraded itself when I connect it directly to the ONT( firmware now is 2.13.4, which is introduced in this July. ). I haven't power on the modem for almost a year. Maybe AT&T auth server start rejects EAP packets from old firmware.
